Library/Formula/camlp5.rb

```diff
@@ -13,20 +16,36 @@ class Camlp5 < Formula
 
   deprecated_option "strict" => "with-strict"
   option "with-strict", "Compile in strict mode"
+  option "with-doc", "Install the html and info documentation"
+  option "with-tex", "Install the pdf, ps, and tex documentation"
 
   depends_on "ocaml"
+  depends_on "gnu-sed" => :build if build.with?("doc") || build.with?("tex")
+  depends_on :tex => [:build, :optional]
+  depends_on "ghostscript" => :build if build.with?("tex")
 
   def install
-    if build.with? "strict"
-      strictness = "-strict"
-    else
-      strictness = "-transitional"
-    end
+    args = ["--prefix", prefix, "--mandir", man]
+    args << "--transitional" if build.without? "strict"
```

Comment:
Their build system defaults to strict mode. Should I also add `args << "--strict" if build.with? "strict" ` in case that default ever changes or is this preferable as it is?